app/models/immunisation_import_row.rb

```diff
@@ -32,6 +32,8 @@ class ImmunisationImportRow
   CARE_SETTING_SCHOOL = 1
   CARE_SETTING_COMMUNITY = 2
 
+  MAX_FIELD_LENGTH = 100
```

Comment:
```suggestion
  MAX_FIELD_LENGTH = 300
```

I think actually we settled on 300 as the length for these kinds of fields, looking at Jake's comment in the Jira ticket.